src/redfishpower/redfishpower.c

```diff
@@ -314,6 +327,59 @@ static void powermsg_init_curl(struct powermsg *pm)
         Curl_easy_setopt((pm->eh, CURLOPT_HTTPGET, 1));
 }
 
+static char *host2ip_url(const char *hostname, const char *path)
+{
+    char *url;
+    char ipstr[INET6_ADDRSTRLEN] = {0};
+    size_t len;
+    struct addrinfo *ai;
+    struct addrinfo *res = NULL;
+    int ret;
```

Comment:
Is this doing a DNS lookup every time a message is sent to a given IP?

Reply:
yeah, i was a little lazy in not doing a cache, since it is a workaround ..... should add a cache?

Reply:
I suppose it's probably no different than what was happening with libcurl (unless it was doing some caching).  Still when doing a status across the whole machine it might hit the DNS pretty hard.  :shrug:  Can always come back later and optimize if there's a problem.

Reply:
current testing showed it was working, which is why i didn't look into optimizing it.  Who knows, maybe this mild slowdown is good given the issue in libcurl :P

Reply:
Ok then :smile: